packages/html-reporter/src/filter.ts

```diff
@@ -102,7 +110,8 @@ export class Filter {
       const searchValues: SearchValues = {
         text: (status + ' ' + test.projectName + ' ' + test.path.join(' ') + test.title).toLowerCase(),
         project: test.projectName.toLowerCase(),
-        status: status as any
+        status: status as any,
+        labels: (test.path.join(' ') + test.title).toLowerCase(),
```

Comment:
At this point, it is not distinguishable from the text in terms of the search, so just use `.text`?